test/core/parser/test_parser.py

```diff
@@ -161,7 +161,10 @@ def testString(self):
         self.check(r'"abc"', String("abc"))
         self.incomplete_error(r'"abc')
         self.check(r'"abc(*def*)"', String("abc(*def*)"))
-        self.check(r'"a\"b\\c"', String(r"a\"b\\c"))
+
+        # The following check is not correct.
+        # the answer *should* be r'a"b\\c"'
+        # self.check(r'"a\"b\\c"', String(r"a\"b\\c"))
```

Comment:
The fix for this should come from the scanner side. Possibly, we'll move tests like this over there.